CPAC/pipeline/engine.py

```diff
@@ -2511,6 +2514,18 @@ def ingress_pipeconfig_paths(cfg, rpool, unique_id, creds_path=None):
                 "",
                 f"{key}_config_ingress",
             )
+        #check if val is a nifti file .nii.gz
+        if val.endswith('.nii.gz'):
+            templates.append([key, val])
+
+    table = check_all_orientations(templates,"RPI")
+    df = pd.DataFrame(table, columns = ['Resource', 'Path', 'Orientation'])
+    
+    # check if any of the values in Orientation column are not RPI
+    if not df[df['Orientation'] != 'RPI'].empty:
+        WFLOGGER.info(f"The following templates are not in RPI orientation: {df}")
+        sys.exit()
```

Comment:
We're just exiting if any template's not in RPI?

Instead of an info message and a generic exit, lets

```Python
from nibabel.orientations import OrientationError
```
at the top and

```suggestion
    other_orientation = df[df["Orientation"] != "RPI"]
    if not other_orientation.empty:
        msg = f"The following templates are not in RPI orientation: {other_orientation}"
        OrientationError(msg)
```

here.